docs/ldml/tr35-keyboards.md

```diff
@@ -1792,16 +1914,17 @@ Used in the `to=`
     **Example**
 
     ```xml
-    <variable id="upper" value="A B CC D E  FF       G" />
-    <variable id="lower" value="a b c  d e  \u{0192} g " />
+    <set id="upper" value="A B CC D E  FF       G" />
+    <set id="lower" value="a b c  d e  \u{0192} g " />
```

Comment:
```suggestion
    <set id="lower" value="a b c  d e  \u{0192} g" />
```

I know that whitespace at the end is ignored, but it's good if the example is clean...